css/properties/max-height.json

```diff
@@ -161,11 +161,11 @@
             "support": {
               "chrome": {
                 "alternative_name": "-webkit-fill-available",
-                "version_added": "28"
+                "version_added": "25"
```

Comment:
I don't understand these changes to this entire file. Can you explain what was inconsistent and how you resolved it?